angular-workspace/projects/ni/nimble-angular/tsconfig.lib.json

```diff
@@ -5,7 +5,6 @@
     "outDir": "../../../out-tsc/lib",
     "declaration": true,
     "declarationMap": true,
-    "inlineSources": true,
```

Comment:
Wasn't super clear but is my understanding correct that this is being removed because it is already specified in the shared root one of the workspace? If so looks good.

Reply:
Well, I moved it to the shared root tsconfig. The `sourceMap` setting was already there, so I figured they should be set together.